### DISCUSSION TITLE
In tabbed mode, titlebars are always visible now

### DISCUSSION THREAD

**Discussion by kartikdc** [i3/i3#5164] [Q&A] [answered]
Since updating to 4.21, titlebars seems to always be visible now even if there is only one application running in the tabbed container.

<details>

<summary>Example screenshot</summary>

![screenshot-20220926042839](https://user-images.githubusercontent.com/77477066/192194135-ace279bd-e210-4978-a867-1b1044636a3c.png)

</details>

Until a few days ago, if a sole window was open inside a tabbed container, the titlebar was not visible. Is there a way to return to the original behaviour?

I checked the changelog but could not find anything except for the new click and drag feature which could be partly relevant.

Thanks all. :) 

---

EDIT:

<details>

<summary>Screenshot after downgrading to 4.20</summary>

![screenshot-20220926043441](https://user-images.githubusercontent.com/77477066/192194681-918578c1-42e9-4e05-93c7-13879575ff3e.png)

</details>

**Comment by orestisfl** [comment, marked as answer]
https://github.com/i3/i3/issues/5149

**Reply by kartikdc** [reply]
Thank you so much
